floris/optimization/load_optimization/load_optimization.py

```diff
@@ -0,0 +1,546 @@
+        wake_slope (float, optional): Wake slope, defined as the lateral expansion of the wake on
+            each side per unit downstream distance along the axial direction. Defaults to 0.3.
+        max_dist_D (flat, optional): Maximum distance downstream of a turbine beyond which wake
+            added turbulence is assumed to be zero, in rotor diameters. Defaults to 10.0
+            (see IEC 61400-1 Ed. 4 Annex E).
+        exp_ws_std (float, optional): Exponent for the wind speed standard deviation.
+            Defaults to 1.0.
+        exp_thrust (float, optional): Exponent for the thrust. Defaults to 1.0.
+
+    Returns:
+        np.array: Array of net revenue for each findex
+
+    """
+
+    revenue = compute_farm_revenue(
+        fmodel=fmodel,
+    )
+
+    farm_voc = compute_farm_voc(
+        fmodel=fmodel,
+        A=A,
+        load_ambient_tis=load_ambient_tis,
+        wake_slope=wake_slope,
+        max_dist_D=max_dist_D,
+        exp_ws_std=exp_ws_std,
+        exp_thrust=exp_thrust,
+    )
+
+    return revenue - farm_voc
+
+
+def find_A_to_satisfy_rev_voc_ratio(
+    fmodel: FlorisModel,
+    target_rev_voc_ratio: float,
+    load_ambient_tis: np.array,
+    wake_slope: float = 0.3,
+    max_dist_D: float = 10.0,
+    exp_ws_std: float = 1.0,
+    exp_thrust: float = 1.0,
+):
+    """Find the value of A that satisfies the target ratio of total farm revenue over all findices
+    to total farm VOC over all findices.
+
+    Args:
+        fmodel (FlorisModel): FlorisModel object
+        target_rev_voc_ratio (float): Target revenue to VOC ratio
+        load_ambient_tis (list or np.array): Ambient 'load' turbulence intensity for each findex,
+            expressed as fractions of mean wind speed
+        wake_slope (float, optional): Wake slope, defined as the lateral expansion of the wake on
+            each side per unit downstream distance along the axial direction. Defaults to 0.3.
+        max_dist_D (flat, optional): Maximum distance downstream of a turbine beyond which wake
+            added turbulence is assumed to be zero, in rotor diameters. Defaults to 10.0
+            (see IEC 61400-1 Ed. 4 Annex E).
+        exp_ws_std (float, optional): Exponent for the wind speed standard deviation.
+            Defaults to 1.0.
+        exp_thrust (float, optional): Exponent for the thrust. Defaults to 1.
+
+    Returns:
+        float: Value of A that satisfies the target revenue to VOC ratio
+
+    """
+
+    # Compute farm revenue
+    farm_revenue = compute_farm_revenue(
+        fmodel=fmodel,
+    )
+
+    # Compute farm VOC
+    farm_voc = compute_farm_voc(
+        fmodel=fmodel,
+        A=1.0,
+        load_ambient_tis=load_ambient_tis,
+        wake_slope=wake_slope,
+        max_dist_D=max_dist_D,
+        exp_ws_std=exp_ws_std,
+        exp_thrust=exp_thrust,
+    )
+
+    return (farm_revenue.sum() / farm_voc.sum()) / target_rev_voc_ratio
+
+
+def find_A_to_satisfy_target_VOC_per_MW(
```

Comment:
I added this function to help when we want to achieve a target VOC/MWh, which is something users might know. It seems like O&M costs are usually report as $/kW capacity/year, but this seems tricky to find unless the set of findices represents 1 year. So VOC/MW/findex seemed like a reasonable alternative (equivalent to VOC/MWh if a findex represents 1 hour of operation).

Reply:
This is helpful, thank you for adding!